features/admin_login.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Admin login
+
+
+    Scenario: Admin tries to log in with correct data
+
+        Given Admin is on the “login” page
+
+        And the account exists
+
+        When Admin entered the correct login and password
+
+        And admin clicks on „login” button
+
+        Then Admin is redirected to the welcome page
+
+        And there is a text „Hello, {username}” at the top of the page, where {username} contains correct admin login.
```

Comment:
Możesz wpisać daną username. Gdyż wiesz jakiego usera masz w aplikacji do testów. To co zastosowałeś sprawdzi się przy pisaniu testów, gdy napiszesz sobie jedną funkcję do logowania i będziesz z niej korzystał.